contrib/mesos/pkg/minion/server.go

```diff
@@ -204,6 +207,18 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args *[]string, log
 		log.Infof("wrote %d bytes to %v", written, logFileName)
 	}()
 
+	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
+	if ms.pathOverride != "" {
+		env := os.Environ()
+		cmd.Env = make([]string, 0, len(env))
+		for _, e := range env {
+			if strings.HasPrefix(e, "PATH=") {
```

Comment:
if PATH is present then the entry in the `env` array should be replaced. if it's not (unlikely), then append it to `env`. we don't want to end up with two `PATH=` entries in the array - that's asking for trouble.